__init__.py

```diff
@@ -31,6 +33,7 @@ def icon():
     return "icon.png"
 def qgisMinimumVersion():
     return "1.7"
+>>>>>>> master
```

Comment:
also this

Reply:
ufff I don't know what happened. There was a conflict that I fixed in this file. I've fixed again.